### PR TITLE
PHPC-1929: ReadConcern tests need not exhaustively test each constant

### PR DESCRIPTION
**Pull request by levon80999**
https://jira.mongodb.org/browse/PHPC-1929

Related to https://jira.mongodb.org/browse/PHPC-1761

### CODE REVIEW

File: tests/readConcern/readconcern-getlevel-001.phpt [hunk 1]
```diff
@@ -5,8 +5,6 @@ MongoDB\Driver\ReadConcern::getLevel()
 
 $tests = [
     null,
-    MongoDB\Driver\ReadConcern::LOCAL,
```
Comment (jmikola) [inline tests/readConcern/readconcern-getlevel-001.phpt:8]: Please either revert the changes to this file or keep at least one usage of a constant (e.g. `MongoDB\Driver\ReadConcern::LOCAL`). Related to my earlier comment, it's helpful for tests to demonstrate actual usage of the classes.

In this case, retaining a single constant here would cover all of the bases for construction:

 * no level
 * a known, supported level (either a literal corresponding to a constant or the constant itself)
 * a known, unsupported level

File: tests/readConcern/readconcern-constants.phpt [hunk 2]
```diff
@@ -9,6 +9,18 @@ var_dump(MongoDB\Driver\ReadConcern::MAJORITY);
...
+    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::LINEARIZABLE),
+    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::LOCAL),
+    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::MAJORITY),
+    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::AVAILABLE),
+    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::SNAPSHOT),
+];
+
+foreach ($tests as $test) {
+    var_dump($test);
+}
```
[the remark this answers is not in the document]
Comment (levon80999) [reply tests/readConcern/readconcern-constants.phpt]: done

File: tests/readConcern/readconcern-getlevel-001.phpt (hunk 1 above)
Comment (levon80999) [reply tests/readConcern/readconcern-getlevel-001.phpt:8]: done

File: tests/readConcern/readconcern-bsonserialize-001.phpt [hunk 3]
```diff
@@ -7,11 +7,7 @@ require_once __DIR__ . '/../utils/basic.inc';
 
 $tests = [
     new MongoDB\Driver\ReadConcern(),
-    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::LINEARIZABLE),
-    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::LOCAL),
-    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::MAJORITY),
-    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::AVAILABLE),
-    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::SNAPSHOT),
+    new MongoDB\Driver\ReadConcern('setTestLevel'),
```
[the remark this answers is not in the document]
Comment (levon80999) [reply tests/readConcern/readconcern-bsonserialize-001.phpt]: done

File: tests/readConcern/readconcern-serialization-001.phpt [hunk 4]
```diff
@@ -38,57 +34,13 @@ object(MongoDB\Driver\ReadConcern)#%d (%d) {
 
 object(MongoDB\Driver\ReadConcern)#%d (%d) {
   ["level"]=>
-  string(12) "linearizable"
+  string(12) "local"
 }
 bool(true)
-C:26:"MongoDB\Driver\ReadConcern":38:{a:1:{s:5:"level";s:12:"linearizable";}}
+C:26:"MongoDB\Driver\ReadConcern":38:{a:1:{s:5:"level";s:12:"setTestLevel";}}
```
Comment (jmikola) [inline tests/readConcern/readconcern-serialization-001.phpt:40]: This test is failing in [Evergreen](https://spruce.mongodb.com/version/63223ccc3e8e865cf15548a1/tasks).

File: tests/readConcern/readconcern-ctor-001.phpt [hunk 5]
```diff
@@ -5,8 +5,6 @@ MongoDB\Driver\ReadConcern construction
 
 var_dump(new MongoDB\Driver\ReadConcern());
 var_dump(new MongoDB\Driver\ReadConcern(null));
-var_dump(new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::LOCAL));
```
Comment (jmikola) [inline tests/readConcern/readconcern-ctor-001.phpt:8]: Please keep one example using a defined constant, as you did in other tests.

**Review by jmikola — CHANGES_REQUESTED** [review]

File: tests/readConcern/readconcern-bsonserialize-001.phpt (hunk 3 above)
[the remark this answers is not in the document]
Comment (jmikola) [reply tests/readConcern/readconcern-bsonserialize-001.phpt]: Related test failure in https://github.com/mongodb/mongo-php-driver/pull/1360#discussion_r972308226

File: tests/readConcern/readconcern-ctor-001.phpt (hunk 5 above)
Comment (levon80999) [reply tests/readConcern/readconcern-ctor-001.phpt:8]: done

**Pull request merged**